modules/microshift-install-rhel-image-mode-csi-vgcleanup.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-csi-vgcleanup_{context}"]
+= Cleaning up container storage for the bootc image and the {microshift-short} CSI
+
+You can remove an additional LVM volume group loop device that was shared with the container and used by the {microshift-short} Logical Volume Manager (LVM) Container Storage Interface (CSI).
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+
+.Procedure
+
+. Clean up the loop device and additional volume group by using the following commands:
++
+.. Detach the loop device by entering the following command:
++
+[source,terminal]
+----
+$ sudo losetup -d "${VGLOOP}"
```

Comment:
This is the response I get from above command

sudo losetup -f "${VGFILE}"
losetup: /var/lib/microshift-lvm-storage.img: failed to set up loop device: No such file or directory

[redhat@microshift-dev ~]$ echo "${VGFILE}"
/var/lib/microshift-lvm-storage.img

Reply:
This is the deletion section, so it looks like your file does not exist. 
Check using `ls` please.

Reply:
tagged in the wrong spot.